src/components/Tooltip.tsx

```diff
@@ -62,6 +62,7 @@ function Tooltip({
                     {...tooltip}
                     {...props}
                     className={classNames('reactist_tooltip', className)}
+                    style={{ zIndex: 100000 }}
```

Comment:
Can / should we make this configurable? Feels strange to ship something with z-index 100000 just because we need in Todoist 😬

Reply:
I'd even go further and say that we do not need to change reactist for this, not even for making it configurable. It should be configurable already. Can't we apply styles to the `.reactist_tooltip` selector from inside a todoist stylesheet?

Reply:
Oops, this was left over from my investigation, this should not be here 😅

Reply:
@gnapse I think we should try to move towards having a consistent `z-index` strategy in Reactist. My worry is that if we don't have a centralized z-index order it tends to get very messy due to certain implementations. How strongly do you feel about keeping this in Todoist?

Reply:
Totally. Ideally we shouldn't ever need to tinker with z-index in app code, and our component library would take care of all that transparently. But until we're in a state where our apps are mostly using basic building blocks from the component library, and only working at a higher level, we may need to still adjust things from time to time.